libqtile/widget/moc.py

```diff
@@ -20,7 +20,7 @@
 import subprocess
 
 
-class Moc(base.ThreadPoolText):
+class Moc(base.InLoopPollText):
```

Comment:
You should probably stay with `ThreadPoolText` (which now that I see it, it is a typo, actually should be `TheradPollText`). The in-loop one should be used for things that are _fast_, since they run in the main event loop, the threaded version is good for things like this.

Rather, you should change the `ThreadPollText` code in `libqtile/widget/base.py` to use the `update_interval` to re-schedule the poll method.